Aftonbladet.js

```diff
@@ -0,0 +1,245 @@
+	"lastUpdated": "2016-08-10 16:45:00"
+}
+
+/*
+	Aftonbladet Translator - Parses Aftonbladet articles and creates Zotero-based metadata.
+	Copyright (C) 2016 Jonatan Svensson Glad
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+function detectWeb(doc, url) {
+	//TODO: adjust the logic here
+	if (url.indexOf('/article')>-1) {
+		return "newspaperArticle";
+	} else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	//TODO: adjust the xpath
+	var rows = ZU.xpath(doc, '//a[contains(@href, "/article")]');
+	for (var i=0; i<rows.length; i++) {
+		//TODO: check and maybe adjust
+		var href = rows[i].href;
+		//TODO: check and maybe adjust
+		var title = ZU.xpathText(rows[i], './/h2');
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = new Array();
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+		newArticle.title = ZU.xpathText(doc, "//h1").replace("\n", " ");	
+		newArticle.date = ZU.xpathText(doc, '//time[@pubdate]');
+		var authors = ZU.xpath(doc, '//address[contains(@class, "abByline")]/div[contains(@class, "abAuthor")]');
+		for (var i=0; i<authors.length; i++) {
+			newArticle.creators.push(ZU.cleanAuthor(authors[i].textContent, "author"));
+		}
+		Z.debug(authors);
+		newArticle.language =  "Swedish";
+		newArticle.publicationTitle = "Aftonbladet";
+		newArticle.ISSN = "1103-9000";
+		newArticle.abstractNote = ZU.xpathText(doc, '//div[@class="abLeadText"]/p/text()');
+		newArticle.location = ZU.xpathText(doc, '//span[@class="abCity"]')
+		/* TODO: This following line needs to be fixed Te replace's causes erroor on e.g. http://tv.aftonbladet.se/abtv/articles/125960 */
+		newArticle.section = ZU.xpathText(doc, '//div[@class="abBreadcrumbs clearfix"]/span[@class="abLeft"]').replace("Startsidan\n/", "").replace("\n/", " /").replace(" / ", "/").replace("Nyheter/", "").replace("Nyheter", "");
```

Comment:
On http://www.aftonbladet.se/nyheter/kolumnister/wolfganghansson/article23315853.ab it needs to remove the last "subsection", since that is the author name. Only occurs on `/kolumnister/`